graphql_compiler/compiler/helpers.py

```diff
@@ -498,6 +498,13 @@ def get_first_folded_edge(self):
         first_folded_edge_direction, first_folded_edge_name = self.fold_path[0]
         return first_folded_edge_direction, first_folded_edge_name
 
+    def at_vertex(self):
+        """Get the Location ignoring its field component."""
```

Comment:
`Location != FoldScopeLocation` here. They are both `BaseLocation` objects, but this does not return a `Location`.